src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresDatatypeIT.java

```diff
@@ -213,6 +216,118 @@ public void testWriteToTableWithUuidColumn() throws Exception {
     }
   }
 
+  @Test
+  public void testWriteToTableWithIntArrayColumn() throws SQLException, InterruptedException {
+    createTableWithIntArrayColumns();
+    connect.configureConnector("jdbc-sink-connector", props);
+    waitForConnectorToStart("jdbc-sink-connector", 1);
+
+    final Schema schema = SchemaBuilder.struct().name("com.example.Person")
+        .field("firstname", Schema.STRING_SCHEMA)
+        .field("lastname", Schema.STRING_SCHEMA)
+        .field("friends", SchemaBuilder.array(Schema.INT32_SCHEMA).build())
+        .field("friendnames", SchemaBuilder.array(Schema.STRING_SCHEMA).build())
+        .build();
+
+    final Struct struct = new Struct(schema)
+        .put("firstname", "Christina")
+        .put("lastname", "Brams")
+        .put("friends", Arrays.asList(10, 6221))
+        .put("friendnames", Arrays.asList("Lucas", "Tom"));
+    produceRecord(schema, struct);
+
+    waitForCommittedRecords("jdbc-sink-connector", Collections.singleton(tableName), 1, 1,
+        TimeUnit.MINUTES.toMillis(2));
+    try (Connection c = pg.getEmbeddedPostgres().getPostgresDatabase().getConnection()) {
+      try (Statement s = c.createStatement()) {
+        try (ResultSet rs = s.executeQuery("SELECT * FROM " + tableName)) {
+          assertTrue(rs.next());
+          assertEquals(struct.getString("firstname"), rs.getString("firstname"));
+          assertEquals(struct.getString("lastname"), rs.getString("lastname"));
+          assertJDBCArray(rs, "friends", struct);
+          assertJDBCArray(rs, "friendnames", struct);
+        }
+      }
+    }
+  }
+
+  @Test
+  public void testWriteToTableWithIntArrayColumnMissingFields() throws SQLException, InterruptedException {
+    createTableWithIntArrayColumnsMissing();
+    props.put(JdbcSinkConfig.AUTO_CREATE, "true");
```

Comment:
auto_create is not required for this testcase. testAutoCreateTableWithPrimitiveTypeArrayColumn should be its own testcase.